packages/create-llama/helpers/types.ts

```diff
@@ -6,6 +6,15 @@ export type TemplateEngine = "simple" | "context";
 export type TemplateUI = "html" | "shadcn";
 export type TemplateVectorDB = "none" | "mongo" | "pg";
 export type TemplatePostInstallAction = "none" | "dependencies" | "runApp";
+export type TemplateDataSource = "none" | "file" | "web";
```

Comment:
@leehuwuj good idea to introduce the datasource type! How about putting type and config in one object? (to simplify parameter passing)
```suggestion
export type TemplateDataSource = {
  type: "none" | "file" | "web";
  config: TemplateDataSourceConfig;
}  
```

Reply:
great suggestion! I'll apply this one. Thank you!